vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,76 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
+import sys
+import csv
+
+
+def csv_to_dict(file_name: str) -> dict:
+    information_dict = {}
+    tmp_dict = {}
+
+    try:
+        # Open the CSV file
+        with open(file_name, 'r') as f:
+            raw = f.readlines()
+
+        header = raw[:1][0].strip().split(',')
+        for _, line in enumerate(raw[1:]):
+            tmp = line.replace("\n", "").split(",")
+            current_key = tmp[0]
+            try:
+                for x, key in enumerate(header):
+                    tmp_dict[key] = tmp[x]
+            except Exception as e:
+                if e is None:
+                    print(e)
+            information_dict[current_key] = tmp_dict
+            tmp_dict = {}
+
+        return information_dict
+
+    except Exception as e:
+        print(e)
+        return {"Error": e}
+
+
+def update_properties(target, new, key):
```

Comment:
Just recommend making these variable names more explicit. you can add types here as well to make it more readable.